src/mbi/dataset.py

```diff
@@ -41,7 +53,7 @@ def __init__(self, data, domain, weights=None):
         self.weights = weights
 
     @staticmethod
-    def synthetic(domain, N):
+    def synthetic(domain, N): # pylint: disable=invalid-name
```

Comment:
Feel free to rename N to "records", just make sure to update all callsites as well if necessary